src/include/sof/audio/module_adapter/iadk/iadk_module_adapter.h

```diff
@@ -144,10 +144,22 @@ int iadk_wrapper_set_configuration(void *md, uint32_t config_id,
 				   const uint8_t *fragment, size_t fragment_size,
 				   uint8_t *response, size_t response_size);
 
+/*
+ * \brief Retrieve module configuration
+ * \param[in] md - struct IadkModuleAdapter pointer
+ * \param[in] config_id - Configuration ID
+ * \param[in] pos - position of the fragment in the large message
+ * \param[in] data_offset_size: size of the whole configuration if it is the first fragment or the
+ *	      only fragment. Otherwise, it is the offset of the fragment in the whole configuration.
+ * \param[in] fragment: configuration fragment buffer
```

Comment:
Here https://github.com/thesofproject/sof/blob/main/src/include/sof/audio/module_adapter/iadk/processing_module_interface.h#L338-L367 `fragment` is described as output.